pandas/core/config_init.py

```diff
@@ -568,6 +568,22 @@ def use_inf_as_na_cb(key):
         validator=is_one_of_factory(["auto", "pyarrow", "fastparquet"]),
     )
 
+
+# Set up the io.orc specific configuration.
+orc_engine_doc = """
+: string
+    The default orc reader/writer engine. Available options:
+    'auto', 'pyarrow', the default is 'auto'
+"""
+
+with cf.config_prefix("io.orc"):
+    cf.register_option(
+        "engine",
+        "auto",
+        orc_engine_doc,
+        validator=is_one_of_factory(["auto", "pyarrow"]),
+    )
```

Comment:
Is it worth adding an option if we currently do not have any plans to add a different engine? 
I would personally only add an option if we actually need one.